src/components/plots/plot.ts

```diff
@@ -23,6 +23,11 @@ module Plottable {
       accessor: Accessor<any>;
       scale?: Scale<D, R>;
     }
+
+    export module Animator {
```

Comment:
Eh I'm going to disagree with this one and say it should be `Plots.Animator.MAIN`. Not much reason other than "it seemed strange when I was going through and refactoring to `Plot.Animator`", though a secondary reason might be avoiding [merging declarations for a class and a module](http://www.typescriptlang.org/Handbook#declaration-merging)